### PR TITLE
Chronos: Fix bigdl-chronos-spark3[all] installation error

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

**Comment by liangs6212** [comment]
@hkvision @TheaperDeng Pls have a look.

File: python/dev/add_suffix_spark2.sh [hunk 1]
```diff
@@ -55,6 +55,10 @@ sed -i "s/bigdl-chronos==/bigdl-chronos-spark2==/g" ${file}
 sed -i "s/bigdl-chronos-spark3==/bigdl-chronos-spark2==/g" ${file}
 sed -i "s/name='bigdl-chronos'/name='bigdl-chronos-spark2'/g" ${file}
 sed -i "s/name='bigdl-chronos-spark3'/name='bigdl-chronos-spark2'/g" ${file}
+sed -i "s/bigdl-orca==/bigdl-orca-spark2==/g" ${file}
+sed -i "s/bigdl-orca-spark3==/bigdl-orca-spark2==/g" ${file}
```
Comment (hkvision) [inline python/dev/add_suffix_spark2.sh:59]: duplicate with line 41/42

Comment (liangs6212) [reply python/dev/add_suffix_spark2.sh]: thx, I will remove duplicate lines.

**Review by TheaperDeng — APPROVED** [review]
LGTM

**Comment by hkvision** [comment]
Also need to modify remove_spark_suffix.sh

**Review by hkvision — APPROVED** [review]

**Comment by TheaperDeng** [comment]
@hkvision should we merge this PR?

**Comment by hkvision** [comment]
Sure 💯

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/view/BigDL-2.0-PR/job/BigDL-Chronos-PR-Validation/598/

**Pull request merged**